src/schema/nmdc.yaml

```diff
@@ -2281,24 +2312,6 @@ slots:
   mod_date:
     range: string
     description: The last date on which the database information was modified.
-  ecosystem_path_id:
```

Comment:
why were all of these slots removed? I have added them back in

Reply:
That was an error/typo.